smoke-test/src/test/resources/cert/horizon/pas_server.crt

```diff
@@ -0,0 +1,27 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
just wondering: aren't the certs already part of the plugin test dependencies?

Reply:
good point, probably we don't need these files at all

Reply:
updated